### PR TITLE
Update Version to 4.4.11.0.0

### PR DESCRIPTION
**Pull request by helium-service**
Adapter version '4.4.11.0.0', partner version '~> 4.11.0'

### CODE REVIEW

**Review by CB-HaoxinLi — DISMISSED** [review]

**Comment by bwised** [comment]
```
WARN  | xcodebuild:  /Users/runner/Library/Developer/Xcode/DerivedData/App-gqtqqrtdhwjjmmeplftoimxhmpgi/Build/Products/Release-iphonesimulator/XCFrameworkIntermediates/UnityAds/UnityAds.framework/Headers/UnityAds.h:39:1: warning: umbrella header for module 'UnityAds' does not include header 'UnityAdsCommon.h'
```

File: .github/workflows/smoke-test.yml [hunk 1]
```diff
@@ -13,3 +13,5 @@ jobs:
     runs-on: macos-14
     steps:
       - uses: chartboost/chartboost-mediation-ios-actions/adapter-smoke-test@v1
+      with:
+            allow-warnings: true
```
Comment (bwised) [inline .github/workflows/smoke-test.yml:17]: Needed to add this due to the above mentioned umbrella header issue.

File: .github/workflows/smoke-test.yml [hunk 2]
```diff
@@ -13,3 +13,5 @@ jobs:
     runs-on: macos-14
     steps:
       - uses: chartboost/chartboost-mediation-ios-actions/adapter-smoke-test@v1
+        with:
+          allow-warnings: true
```
Comment (bwised) [inline .github/workflows/smoke-test.yml:17]: Needed to add this due to the above mentioned umbrella header issue.

**Review by daniel-barros — APPROVED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -13,3 +13,5 @@ jobs:
     runs-on: macos-14
     steps:
       - uses: chartboost/chartboost-mediation-ios-actions/release-adapter@v1
+        with:
+          allow-warnings: true
```
Comment (bwised) [inline .github/workflows/release.yml:17]: Had to do it here as well in order to release.

**Review by daniel-barros — APPROVED** [review]

**Pull request merged**